src/App.java

```diff
@@ -1,34 +1,264 @@
+// https://docs.oracle.com/javase/8/docs/api/java/util/InputMismatchException.html
+import java.util.InputMismatchException;
 import java.util.Scanner;
 
 public class App {
-    
-    /**
-     * This method is the primary driver method for your program.
-     * It should primarily call other methods to do its work.
-     * Do NOT edit this method until after you have fully satisfied the reqs for a 5
-     */
 
+    // Initialize scanner & variables
+    Scanner madlibs = new Scanner(System.in);
+    int choice;
+
+    // 14-17 learned from https://www.geeksforgeeks.org/how-to-print-colored-text-in-java-console/
+    // and https://codehs.com/tutorial/ryan/add-color-with-ansi-in-javascript
+    // colors used throughout the program
+    String g = "\u001B[32m";
+    String r = "\u001B[31m";
+    String b = "\u001B[34m";
+    String w = "\u001B[37m";
+    
+    // runs the main program calling on other methods
```

Comment:
In the future, use comments of the form `/** Documentation */` for documentation comments before methods. Java and VS code interpret those comments as specific code hints and uses them for prompts in a way they do not with standard `//` comments